Training/train.sh

```diff
@@ -2,24 +2,41 @@
 
 #The private IPs of the AWS instances to be used for training the model.
 #List the local instance (the one on which this script is running) first.
-#The format is ("instance1" "instance2", ...)
-REMOTE_HOSTS=("172.31.61.99" "172.31.59.203")
+#The format is ("instance1" "instance2" ...)
+#HOSTS=("172.31.61.99" "172.31.59.203")
+#HOSTS=("172.31.61.99")
+HOSTS=("172.31.59.203")
 
 #The number of GPUs to use on each instance.
 #Use the same order as for HOSTS.
-GPUS=(1 1)
+GPUS=(1)
 
 #The location of the data for training.
-export DATA_FOLDER="/home/ubuntu/Insight/s3mnt/data2/preprocessed/"
+export TRAINING_DIRECTORY="/home/ubuntu/Insight/s3mnt/Training_Soft/"
+
+#The location of the data for testing/validation.
+export TEST_DIRECTORY="/home/ubuntu/Insight/s3mnt/Test_Soft_Binary/"
 
 #The location of the model to train
-export MODEL="/home/ubuntu/Insight/Vulnerable_Buildings/Models/inception_model.h5"
+export MODEL_FILENAME="/home/ubuntu/Insight/s3mnt/Models/BinaryFocalLoss/soft_model.h5"
```

Comment:
Again, these paths are too specific for your case. 

Replace these statements with something that relates to your current working directory. For example

```
export PYTHONPATH=$PYTHONPATH:`pwd`
```
Get's the current directory in bash style. 

Generally, using bash scripts this way is not a good way. Try to put it into a python file instead `train.py` and then use command line arguments with good defaults to add (optional) parameters. For example, this is one of my [`train.py`](https://github.com/AntonMu/TrainYourOwnYOLO/blob/master/2_Training/Train_YOLO.py)